.github/workflows/apply-terraform.yml

```diff
@@ -135,12 +138,20 @@ jobs:
         ENVIRONMENT: ${{ inputs.ENVIRONMENT }}
         APP_NAME: ${{ inputs.APP_NAME }}
         INFRASTRUCTURE_FOLDER: ${{ inputs.INFRASTRUCTURE_FOLDER }}
+        ECR_REPO: ${{ inputs.ECR_REPO }}  
         IMAGE_TAG: ${{ inputs.IMAGE_TAG }}
       run: |
         if [[ $INFRASTRUCTURE_FOLDER ]]; then
           cd main
           if [[ $IMAGE_TAG ]]; then
             make docker_update_tag tag=$ENVIRONMENT IMAGE_TAG=$IMAGE_TAG
+          elif [[ $ECR_REPO ]]; then
+            echo "Processing $ECR_REPO"
+            IFS=',' read -ra ECR_REPO <<< "$ECR_REPO"
```

Comment:
it'll be worth documenting that this is a going to require a comma separated string. Also do we know what the expected outcome is if there is only one repository and no comma seperation?

Reply:
I have tested the scenario where there is only one value in $ECR_REPO. It worked as expected, executing the loop only once.

Reply:
I have documented that `$ECR_REPO `is a comma-delimited string.
https://github.com/i-dot-ai/i-dot-ai-core-github-actions/blob/a9f4404dbc39396d80e55e07786abe77c4f421e7/.github/workflows/apply-terraform.yml#L38